mkdocs.yml

```diff
@@ -129,6 +129,9 @@ nav:
         - "Heap allocation"                                                      : allocation.md
         - "Garbage collection"                                                   : gc_overview.md
         - "Garbage collection policies"                                          : gc.md
+        - "Troubleshooting garbage collection" :
```

Comment:
Looking at how this displays in the TOC, I think we can do better across both of our topic areas to avoid titles getting truncated.

How about:

- "Heap allocation"
- "Garbage collection (GC)"
- "GC policies"
- "Troubleshooting GC"
- "Verbose GC logs"
- "Log examples"

We can just change this here in the mkdocs.yml without modifying the `<h1>` in the topics themselves.

Reply:
This is nice.